_infra/helm/templates/service.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: samplesvc
+spec:
+  type: ClusterIP
+  ports:
+  - port: {{ .Values.service.port }}
+    protocol: TCP
+    targetPort: {{ .Values.service.port }}
+  selector:
+    app: {{ include "common.name" . }}
```

Comment:
there will be no common dependency so this can be removed, aswell as the selector